src/cluster-legacy.c

```diff
@@ -0,0 +1,6417 @@
+/* Redis Cluster implementation.
```

Comment:
if we do file renames in a separate commit from mass code rename, and copy-paste code moves, it shouldn't matter for git (it'll be able to understand where the code went to).
@jhershberg please ack you did that.

on the other hand, if the same commit splits one file into 3, then git won't be able to realize anything about the rename (and it doesn't matter which one will be named cluster.c)

Reply:
I changed it so that first cluster.c is first renamed to cluster-legacy.c in the first commit. Now "git blame" on cluster-legacy.c shows full history.